x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/LookupFromIndexIT.java

```diff
@@ -68,14 +69,60 @@
 import static org.elasticsearch.test.MapMatcher.assertMap;
 import static org.hamcrest.Matchers.empty;
 
+@TestLogging(value = "org.elasticsearch.compute.operator.lookup:TRACE", reason = "debugging")
```

Comment:
Should this be removed/commented?

Reply:
Either or. You can commit these if you are worried about things failing. But at this point I'm not sure it's good to zap, yeah.